sidebars.js

```diff
@@ -327,6 +327,7 @@ module.exports = {
         "server/erlangSDK",
         "server/rustSDK",
         "server/cppSDK",
+        "server/deprecation-notices",
```

Comment:
dont we have a note on "why is this async" inline?  We should update that for the specific SDKs that have the sync APIs